src/framework/mlt_repository.c

```diff
@@ -98,12 +98,33 @@ mlt_repository mlt_repository_init( const char *directory )
 	free(newpath);
 #endif
 
+	mlt_tokeniser tokeniser = mlt_tokeniser_init();
+	int dl_length = mlt_tokeniser_parse_new( tokeniser, getenv( "MLT_REPOSITORY_DENY" ), ":" );
```

Comment:
This new environment variable should be documented via a doxygen comment in `mlt_repository.h` or `mlt_factory.h` to make it show up [here](https://www.mltframework.org/doxygen/envvars.html). 
Here is how that is done:
` * \envvar \em MLT_REPOSITORY overrides the default location of the plugin modules, defaults to \p PREFIX_LIB.
`
Thanks